spec/lib/capistrano/doctor/environment_doctor_spec.rb

```diff
@@ -21,8 +21,8 @@ module Doctor
 
       describe "Rake" do
         before do
-          load File.expand_path("../../../../../lib/capistrano/doctor.rb",
-                                __FILE__)
+          $:.unshift File.expand_path('../../../../../lib', __FILE__)
```

Comment:
Do we need this line at all? I can't test in the Debian packager but at least for me it is not necessary (the specs still pass without it). Because we are already adding ../lib to the load path here:

https://github.com/capistrano/capistrano/blob/1bfbe82007380594358771021ba7a0fa4c2deff9/spec/spec_helper.rb#L1